control/controlclient/auto.go

```diff
@@ -596,10 +600,15 @@ func (c *Client) mapRoutine() {
 	}
 }
 
+// AuthCantContinue() returns true if the auth state machine is ready to try
```

Comment:
Go style is without drop parens. More nitpicky, but: Go style is also to say "reports whether" instead of "returns true if".